src/serializer/serializer.js

```diff
@@ -181,20 +175,7 @@ export class Serializer {
   serializedValues: Set<Value>;
   serializedScopes: Map<DeclarativeEnvironmentRecord, ScopeBinding>;
   capturedScopeInstanceIdx: number;
-  varsExpression: BabelNodeIdentifier;
-  nextVarsIndex: number;
-  availableVars: Array<BabelNodeMemberExpression>;
-
-  allocateIndexedVar(): BabelNodeMemberExpression {
-    if (this.availableVars.length > 0) {
-      return this.availableVars.pop();
-    }
-    return t.memberExpression(this.varsExpression, t.numericLiteral(this.nextVarsIndex++), true);
-  }
-
-  releaseIndexedVar(indexedVar: BabelNodeMemberExpression) {
-    this.availableVars.push(indexedVar);
-  }
+  capturedScopes: BabelNodeIdentifier;
```

Comment:
Perhaps call this capturedScopes(Map/Array)? Just reading this code I can't figure out what is stored here.